dubbo-filter/dubbo-filter-cache/src/main/java/org/apache/dubbo/cache/support/expiring/ExpiringMap.java

```diff
@@ -84,6 +84,11 @@ public V put(K key, V value) {
     public V get(Object key) {
         ExpiryObject object = delegateMap.get(key);
         if (object != null) {
+            long timeIdle = System.currentTimeMillis() - object.getLastAccessTime();
+            if (timeIdle >= expireThread.getTimeToLive() * 1000) {
```

Comment:
shall we check if `expireThread.getTimeToLive()` is zero or even negative

Reply:
oh yes. we should be